sv.h

```diff
@@ -746,6 +746,40 @@ Unsets the PV status of an SV.
 Tells an SV that it is a string and disables all other C<OK> bits.
 Will also turn off the UTF-8 status.
 
+=for apidoc Am|U32|SvPIOK|SV* sv
+Returns a bool indicating whether the SV is both C<SvPOK()> and
+C<SvIOK()> at the same time. Equivalent to C<SvIOK(sv) && SvPOK(sv)> but
+more efficient.
+
+=for apidoc Am|void|SvPIOK_on|SV* sv
+Tells an SV that is a string and a number in one operation. Equivalent
+to C<SvIOK_on(sv); SvPOK_on(sv);> but more efficient.
+
+=for apidoc Am|void|SvPIOK_off|SV* sv
+Unsets the PV and IV status of an SV in one operation. Equivalent to
+C<SvIOK_off(sv); SvPK_off(v);> but more efficient.
+
+=for apidoc Am|bool|PIOK_sv_isbool|SV* sv
+Checks if a PIOK sv is a bool. Note that it is the callers
+responsibility to ensure that the sv is SvPIOK. This macro is only
+useful in specialized logic where the flags have already been checked to
+be correct. Almost always you should be using the function
+C<sv_isbool(sv)> instead.
+
+=for apidoc Am|bool|PIOK_sv_isbool_true|SV* sv
+Checks if a PIOK sv is a true bool. Note that it is the callers
+responsibility to ensure that the sv is SvPIOK. This macro is only
+useful in specialized logic where the flags have already been checked to
+be correct. This is NOT what you should use to check if an SV is "true",
+for that you should be using the function C<SvTRUE(sv)> instead.
```

Comment:
s/function/macro/ perhaps? I don't know if we usually bring attention to how many of our "API functions" are in fact macros.

Reply:
tried to avoid both terms in all the examples I checked. I think there was one case I couldnt avoid tho. (i forget which, sleepy now)